libpod/volume.go

```diff
@@ -68,3 +76,57 @@ func (v *Volume) Scope() string {
 func (v *Volume) IsCtrSpecific() bool {
 	return v.config.IsCtrSpecific
 }
+
+// Mount the volume
+func (v *Volume) Mount() error {
+	if v.MountPoint() == "" {
+		return errors.Errorf("missing device in volume options")
+	}
+	mounted, err := mount.Mounted(v.MountPoint())
+	if err != nil {
+		return errors.Wrapf(err, "failed to determine if %v is mounted", v.Name())
+	}
+	if mounted {
+		return nil
+	}
+	options := v.Options()
+	if len(options) == 0 {
+		return errors.Errorf("volume %v is not mountable, no options available", v.Name())
+	}
+	mountOpts := options["o"]
+	device := options["device"]
+	if options["type"] == "nfs" {
```

Comment:
This should not be an option. Volume has a "type" field, and that is what we should be using.

Reply:
Correction, I meant `Driver` - I think I'd count NFS mounts as a type of volume driver...

Or we should add a type field. Let's not stuff critical information into maps. Make this top-level.

Reply:
We can add this under the covers, but if we want to follow the Docker CLI, we have to allow users to specify nfs in the manner.

Reply:
Sure, matching their CLI is fine, but let's not let the CLI determine how we set up internal data structures